api-test/src/main/resources/logback.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+
+    Copyright (C) 2023 Dremio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<configuration debug="true">
+  <property name="catalog.migration.log.dir" value="${catalog.migration.log.dir:-logs}"/>
```

Comment:
You still write files to `api/logs`

Reply:
I am saying, that the log files _still_ leak to `api/logs/` - maybe just run it locally and you'll see it?

Reply:
Yes, at the time of comment `CatalogMigratorParamsTest` was not having that system property. Hence, it was leaking. 
With the latest changes, I have verified that there is no leak by executing `gradle clean build` and checking for the log file.